src/utils/llms.py

```diff
@@ -1,6 +1,8 @@
 from dotenv import load_dotenv
 from langchain_anthropic import ChatAnthropic
 from langchain_google_genai import ChatGoogleGenerativeAI
+from langchain_groq import ChatGroq
```

Comment:
I propose to skip this. We are not likely going to use it, and it gave me an error because i did not have an api key. Change also in review PR.